scripts/dns-delay/install-dns-delay.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Do you need the sysctl command too "sysctl -w net.core.default_qdisc=fq_codel" for GCE?

Reply:
Good catch, I was omitting it initially just to make sure the rest works. I will add it.

Reply:
Added now.